webapp/tbapi/tbapi/templates/account/authorize.html

```diff
@@ -5,9 +5,11 @@
   <form action="" method="post" class='v-center'>
     <div style='width: 100%'>
       {{ render_form(form) }}
+
       <div role="toolbar" class="signin-toolbar btn-toolbar">
-        <button class="btn btn-block btn-primary btn-lg">Registrarse</button>
+        <button class="btn btn-secondary btn-lg col-xs-4" onClick="javascript:history.back()">Atrás</a>
```

Comment:
Not a huge fan of inline javascript and worried this might lead to a redirection loop.

Would you be able to use a normal link (ie `<a>`) to the react host (`REACT_APP_API_PATH`) instead?